examples/realtime.rs

```diff
@@ -60,12 +85,22 @@ fn main() {
         if dt >= Duration::from_secs(DURATION_SECS as u64) {
             break;
         }
+
+        // Access our Spatial Controls
+        let mut spatial_control = signal.control::<oddio::Spatial<_>, _>();
+
         // This has no noticable effect because it matches the initial velocity, but serves to
         // demonstrate that `Spatial` can smooth over the inevitable small timing inconsistencies
         // between the main thread and the audio thread without glitching.
-        signal.control::<oddio::Spatial<_>, _>().set_motion(
-            [-speed + speed * dt.as_secs_f32(), 10.0, 0.0].into(),
-            [speed, 0.0, 0.0].into(),
+        spatial_control.set_motion(
+            [-SPEED + SPEED * dt.as_secs_f32(), 10.0, 0.0].into(),
+            [SPEED, 0.0, 0.0].into(),
         );
+
+        // We also could adjust the Gain here in the same way:
+        let mut gain_control = signal.control::<oddio::Gain<_>, _>();
+
+        // Just leave the gain at it's basic volume. (sorry this can be a bit loud!)
```

Comment:
```suggestion
        // Just leave the gain at its natural volume. (sorry this can be a bit loud!)
```
Feel free to adjust the `* 80.0` where the signal is built down to something saner; I've been testing on broken speakers :D